ci-operator/step-registry/openstack/conf/clouds/openstack-conf-clouds-commands.sh

```diff
@@ -62,17 +62,20 @@ new_application_credentials() {
 # Setting a default authentication method based on version number
 if [[ "$OPENSTACK_AUTHENTICATION_METHOD" == "version-default" ]]; then
 	if is_openshift_version_gte "4.13"; then
+		info 'Detected version gte 4.13: setting application credentials authentication.'
 		OPENSTACK_AUTHENTICATION_METHOD='application-credentials'
 	else
+		info 'Detected version lt 4.13: setting password authentication.'
 		OPENSTACK_AUTHENTICATION_METHOD='password'
 	fi
 fi
 
-# Skip application credentials when testing old OpenShift versions, as appcreds
-# are only supported in OCP v4.12+.
-if ! is_openshift_version_gte "4.12"; then
-	info 'The detected version does not support application credentials.'
-	OPENSTACK_AUTHENTICATION_METHOD='password'
+# Setting a default authentication method based on version number
+if [[ "$OPENSTACK_AUTHENTICATION_METHOD" == "application-credentials" ]]; then
+	if ! is_openshift_version_gte "4.13"; then
```

Comment:
This now prevents testing application credentials on 4.12, which should work for the most part except for the CSI migration. Are we sure we want this? If that's the case, this should be highlighted in the commit message, as we're changing the current behavior.

Reply:
I believe that we should remove this check entirely, and let the users deal with consequences when they explicitly set the auth method to `application-credentials`. A hard crash is for me the next best option, and I have no strong opinion on the version we want to apply this to. Shall I set 4.12?